### PR TITLE
Remove the "Extending with Compiler Extensions" Learn Page for SLP5

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Remove the "Extending with Compiler Extensions" Learn Page for SLP5.
> Fixes #

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: .github/workflows/add_link_checker.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        uses: peter-evans/link-checker@v1
+      - name: Create Issue From File
+        uses: peter-evans/create-issue-from-file@v2
+        with:
+          title: Link Checker Report
+          content-filepath: ./link-checker/out.md
+          labels: report, automated issue
+          assignees: praneesha
+      - name: Fail if there were link errors
+        run: exit ${{ steps.lc.outputs.exit_code }}
```
Comment (anupama-pathirage) [inline .github/workflows/add_link_checker.yml:20]: Is this intended to have this along with removing the `Extending with compiler extensions` page?

File: .github/workflows/add_spell_checker.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# This is the config of the spell checker GitHub Action (i.e., https://github.com/marketplace/actions/check-spelling).
+
+on: [pull_request]
+name: Check spelling
+jobs:
+  spellChecker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check spelling
+      - uses: check-spelling/check-spelling@0.0.16-alpha
```
Comment (anupama-pathirage) [inline .github/workflows/add_spell_checker.yml:10]: Same as above comment?

File: js/redirections.js [hunk 3]
```diff
@@ -237,5 +237,6 @@ let redirections = {
     "/1.1/learn/by-example/readonly-objects.html":"/1.1/page-not-available.html",
     "/1.0/learn/by-example/readonly-objects.html":"/1.0/page-not-available.html",
     "/swan-lake/learn/api-docs/ballerina/*/objects/*":"/swan-lake/learn/api-docs/ballerina/*/classes/*",
-    "/swan-lake/learn/observing-ballerina-code/":"/swan-lake/page-not-available.html"
+    "/swan-lake/learn/observing-ballerina-code/":"/swan-lake/page-not-available.html",
```
Comment (anupama-pathirage) [inline js/redirections.js:240]: Why `observing ballerina code` page is removed in this PR?

File: learn.md [hunk 4]
```diff
@@ -29,7 +29,7 @@ redirect_from:
    <p>Download Ballerina, set the tools up, and take the Quick Tour.</p>
 
    <ul class="cLearnLandingLinks">
-   <li><a href="/learn/installing-ballerina/" class="cGreenLinkArrow">Setting up Ballerina</a></li>
+   <li><a href="/installing-ballerina/" class="cGreenLinkArrow">Setting up Ballerina</a></li>
```
Comment (anupama-pathirage) [inline learn.md:32]: Why do we remove `/learn/` from here?

**Comment by praneesha** [comment]
Closing as this PR contains changes in invalid files. Instead, the changes required in this are done via https://github.com/ballerina-platform/ballerina-dev-website/pull/1418.

**Pull request closed without merging**